examples/chart/teleport-kube-agent/templates/config.yaml

```
@@ -96,10 +102,13 @@ data:
         {{- end }}
         {{- toYaml .Values.databases | nindent 6 }}
       {{- end }}
+      {{- if .Values.databaseResources }}
+      resources:
+        {{- toYaml .Values.databaseResources | nindent 6 }}
```

Not sure, but I think this one should be
```suggestion
        {{- toYaml .Values.databaseResources | nindent 8 }}
```